vmangos/core
# Gnomeregan - convert script texts to broadcast texts

## 🍰 Pullrequest
- convert all texts to broadcast texts
- correct some chat types during Blastmaster Emi Shortfuse event
- fix Grubbis not doing his intro text

### Proof
- None

### Issues
- None

### How2Test
- check texts during Blastmaster Emi Shortfuse event
- check texts during escort quest
- check texts and sounds during Mekgineer Thermaplugg fight

### Todo / Checklist
